src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -611,7 +612,7 @@ private Dictionary<OperationType, bool> GetConfiguredRestOperations(string entit
 
             if (dbObject.SourceType == EntitySourceType.StoredProcedure)
             {
-                Entity entity = _runtimeConfig.Entities[entityName];
+                Entity entity = _runtimeConfigProvider.GetConfig().Entities[entityName];
```

Comment:
This is another location where you are mid-procedure and pulling a new version of the runtimeconfig. This presents opportunity for different sections of OpenAPI doc to contain contents of two different runtimeconfig objects.

NOTE: It may not be appropriate to pass the entire runtimeconfig object. Looking at this code, can you see what object/element from the runtimeconfig is needed here?